src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/config/SwaggerPublisher.java

```diff
@@ -20,26 +20,31 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
```

Comment:
Can we update to refere to workflow rather than travis: .github/workflows/swagger.yml

Reply:
commented updated

Reply:
Resolved in e9411a965550fa4fb436d81c7730e64acf15fbcd